handlers/github_organization_repo_create_v1/handler/init.rb

```diff
@@ -37,17 +37,25 @@ def execute()
         raise StandardError.new "org_name must be defined as an info_value or a parameter."
       end
 
-      url = "https://api.github.com/orgs/#{org_name}/repos?access_token=#{@info_values['access_token']}"
+      url = "https://api.github.com/orgs/#{org_name}/repos"
       
       resource = RestClient::Resource.new(url)
 
       data = {name: @parameters['name'], description: @parameters['description']}
 
       data['private'] = true if @parameters['visibility'].downcase == "private"
 
-      puts "Calling URL https://api.github.com/orgs/#{org_name}/repos?access_token=XXX" if @debug_logging_enabled
-      response = resource.post(data.to_json, {content_type: :json})
+      puts "Calling URL https://api.github.com/orgs/#{org_name}/repos" if @debug_logging_enabled
+
       response = nil
+      # create repo
+      response = resource.post(
+        data.to_json, 
+        {
+          content_type: :json,
+          authorization: "token #{@info_values['access_token']}"
+        })
+      puts response
```

Comment:
Remove this or use  `if @debug_logging_enabled`

Reply:
Sorry, leftover from testing.  I'll remove it.